providers/keycloak/src/airflow/providers/keycloak/auth_manager/resources.py

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from enum import Enum
+
+
+class KeycloakResource(Enum):
```

Comment:
Are these resources different per auth manager? Shouldn't they all be working with the same resources?

Reply:
As in each Auth Manager may use a subset of the resources? Or they may make up entirely new ones?

Reply:
>  To do that I use this enum, the value of each enum (e.g. Asset) will be sent to the Keycloak server and that identifier will be compared to resources defined in the Keycloak realm.

Ahh, interesting, it would be nice if each auth manager used the same set of string for each method and then they could just be defined in one place and imported into each auth manager. But the precedent is already set! All good from me :+1:

Reply:
It is hard to assume that all auth managers will use the same strings. For instance, FAB auth manager uses its own set of strings. Simple auth manager does not use any given its simplicity.